internal/converter/converter.go

```diff
@@ -180,6 +182,38 @@ func ConvertWithOptions(req *pluginpb.CodeGeneratorRequest, opts options.Options
 	}, nil
 }
 
+func mergeTags(tags []*base.Tag) []*base.Tag {
+
+	if len(tags) == 0 {
+		return tags
+	}
+
+	res := make([]*base.Tag, 0, len(tags))
+	found := make(map[string]*base.Tag)
+
+	for _, tag := range tags {
+		if found[tag.Name] == nil {
+			found[tag.Name] = tag
+			res = append(res, tag)
+			continue
+		}
+
+		if tag.Description != "" {
+			found[tag.Name].Description = tag.Description
```

Comment:
@sudorandom, I'm using the map reference below here. Can you look again?

Reply:
Oh! I understand now. Sorry about that!